pointcloud_filter_node:
  ros__parameters:
    x_min: 0.0
    x_max: 7.0
    y_min: -3.0
    y_max: 3.0
    z_min: 0.0
    z_max: 4.0 
    board_width: 1.8 